pkg/backend/httpstate/client/client.go

```diff
@@ -40,6 +42,12 @@ import (
 	"github.com/pulumi/pulumi/pkg/workspace"
 )
 
+// policyPackVersionTagPattern is the regex pattern for allowable version tags
+// of a Policy Pack.
+const policyPackVersionTagPattern = "^[a-zA-Z0-9-_.:]{1,150}$"
```

Comment:
Why did you settle on this particular regular expression? Is it designed to match NPM or some other common package manager? If not, I'd suggest we use the more strict `"^[a-zA-Z0-9-_.]{1,100}$"` that we use for most other user-provided identifiers in Pulumi, like stack names.

We can always relax the naming requirements later. (But having a tag 100+ characters in length seems like it could only lead to problems, IMHO.)

Reply:
I was matching some other regex. Happier to go with stricter regex. 

- [ ] I will also update the regex in the service.